views/verify_email.py

```diff
@@ -0,0 +1,68 @@
+from rest_framework import generics, permissions
```

Comment:
Can you please separate the import statements?
1. `django` and `rest_framework` dependencies.
> a line break
2. omniport dependencies
> a line break
3. app dependencies